updating/updating-cluster-between-minor.adoc

```diff
@@ -17,6 +17,12 @@ Use the web console or `oc adm upgrade channel _<channel>_` to change the update
 * Have access to the cluster as a user with `admin` privileges.
 See xref:../authentication/using-rbac.adoc[Using RBAC to define and apply permissions].
 * Have a recent xref:../backup_and_restore/backing-up-etcd.adoc#backup-etcd[etcd backup] in case your upgrade fails and you must xref:../backup_and_restore/disaster_recovery/scenario-2-restoring-cluster-state.adoc#dr-restoring-cluster-state[restore your cluster to a previous state].
++
+{product-title} 4.9 requires an upgrade from etcd version 3.4 to 3.5. If the etcd Operator halts the upgrade, an alert displays. To clear this alert, restart the upgrade using the force flag.
+----
```

Comment:
* This should have `[source,terminal]` above to define the source language
* You'll also need a + on the line above this (and above the source language line), so that this code block is indented properly with the bullet